src/PhpSpec/Console/Application.php

```diff
@@ -57,9 +58,17 @@ public function getContainer()
      */
     public function doRun(InputInterface $input, OutputInterface $output)
     {
+        $helperSet = $this->getHelperSet();
         $this->container->set('console.input', $input);
         $this->container->set('console.output', $output);
-        $this->container->set('console.helper.dialog', $this->getHelperSet()->get('dialog'));
+        $this->container->set('console.helper.dialog', $helperSet->get('dialog'));
```

Comment:
OK. In that case I'm going to break BC and remove dialogHelper here.

It annoys me supporting `ask` and `askAndValidate` - part of me thinks they should be removed and Extension writers can catch up, but that might be too big a step until the versioning policy is more explicitly stated.